app/views/shared/partials/_revealing_checkbox.html.erb

```diff
@@ -9,7 +9,7 @@
     <% if error %>
       <% error_class='govuk-form-group--error'
          error_id = "#{model.model_name.to_s.underscore.dasherize}-#{name.dasherize}#{number}" %>
-      <p class='govuk-error-message' id="<% error_id %>"><%= error %></p>
+      <p class='govuk-error-message' id="error_id"><%= error %></p>
```

Comment:
I'm not sure this quite works. This will set the id to the string "error_id" rather than the contents of the `error_id` variable, which I think will cause issues if there's more than one error on the page. 

Would `id="<%= error_id %>"` work instead? [This](https://github.com/Shopify/better-html#syntax-restriction) seems to suggest it should be allowed.